modele-social/règles/salarié.yaml

```
@@ -159,43 +161,152 @@ contrat salarié . frais professionnels . titres-restaurant . contrôle taux emp
-  formule:
+contrat salarié . frais professionnels . transports personnels . proportion déduction:
+  titre: Facteur de proportion de la déductibilité
+  valeur:
     produit:
-      assiette: distance mensuelle
-      facteur: 0.25 €/km
+      assiette:
+        le minimum de:
+          - temps de travail . quotité de travail
+          - 50%
+      taux: 200%
+  références:
+    Article R3261-14 du code du travail, version 11/05/2020: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000041865023/2020-05-11/
 
-contrat salarié . frais professionnels . indemnité kilométrique vélo . part déductible:
-  titre: indemnité kilométrique vélo (déductible)
-  formule:
+contrat salarié . frais professionnels . transports personnels . carburant faible émission:
+  valeur: oui
+
+contrat salarié . frais professionnels . transports personnels . carburant faible émission . montant:
+  titre: Prise en charge du carburant pour véhicules électriques, hybrides rechargeables ou hydrogènes
+  question: Quel montant l'employeur prend-il en charge des dépenses en carburant pour véhicules électriques, hybrides rechargeables ou hydrogènes ?
+  unité: €/mois
+  par défaut: 0 €/mois
+  description: |
+    L'employeur peut prendre en charge tout ou partie des frais de carburant dépensés par l'employé pour son véhicule électriques, hybrides rechargeables ou hydrogènes, sur présentation de justificatif.
+
+    Cette prise en charge peut profiter d'une exonération des cotisations sociales et de l'impôt sur le revenu. Le montant maximal déductible est de 200€/mois, mais attention
+
+      - le plafond est partiellement réduit du montant de la prise en charge des frais d'abonnement aux transports publics
+
+      - cette prise en charge de carburant entre dans la même assiette que la prise en charge du forfait mobilités durables.
+
+    Dans le cas d'un temps partiel, l'avantage sera le même pour un mi-temps ou plus. En dessous, un facteur proportionnel sera appliqué.
+
+    Pour verser une prime de salaire équivalente à 200€/mois à son salarié sans ce dispositif, **l'employeur devrait débourser près de 500€ pour un salaire médian**.
+  références:
+    Articles R3261-11 à -13 du code du travail, version 11/05/2020: https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006072050/LEGISCTA000018487476/2020-05-11
+    Article 81 du code des impôts, version en vigueur au 31/12/2020: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000042910732/2020-12-31/
+
+contrat salarié . frais professionnels . transports personnels . carburant faible émission . part déductible:
+  titre: Prise en charge du carburant pour véhicules électriques, hybrides rechargeables ou hydrogènes (part déductible)
+  unité: €/mois
+  valeur:
     valeur: montant
-    plafond: 200 €/an
+    plafond:
+      valeur:
+        le maximum de:
+          - 0
+          - proportion déduction * 500€/mois - abonnement transports publics . prise en charge
+      plafond: proportion déduction * 200€/mois
```

```suggestion
  plafond:
    le minimum de:
       - proportion déduction * 200€/mois
       - valeur: proportion déduction * 500€/mois
         abattement: abonnement transports publics . prise en charge
```

L'abatttement est a une borne inférieur à 0, par conséquent, pas besoin de rajouter un maximum. Par ailleurs, plutôt que d'enchaîner les plafonds, je propose une écriture avec un minimum pour plus de clarté.